app/templates/settings/users/index.hbs

```diff
@@ -59,6 +64,15 @@
                       {{membership.organization.name}}
                     </p>
                   </div>
+                  {{#if membership.organization.isBlocked}}
+                    <AuPill @icon="circle-x" @skin="error" @size="small">
+                      {{t "blocked-organization"}}
```

Comment:
Wasn't the conclusion of the discussion that organization block results in blocking the membership hence no additional pill required here?